app/src/main/java/net/gsantner/markor/model/Document.java

```diff
@@ -61,16 +61,9 @@ public class Document implements Serializable {
 
     public Document(@NonNull final File file) {
         _file = file;
-        final String name = _file.getName();
-        final int doti = name.lastIndexOf(".");
-        if (doti < 0) {
-            _fileExtension = "";
-            _title = name;
-        } else {
-            _fileExtension = name.substring(doti).toLowerCase();
-            _title = name.substring(0, doti);
-        }
         _path = _file.getAbsolutePath();
+        _title = FileUtils.getFilenameWithoutExtension(_file);
+        _fileExtension = FileUtils.getFilenameWithoutExtension(_file);
```

Comment:
wrong method called

also don't need to be a variable anymore, should be fine to always calculate it from filename member